src/charm.py

```diff
@@ -248,8 +248,8 @@ def configure_workload_container(self, event: ConfigChangedEvent) -> None:
             msg = "Updating Nginx site config"
             logger.info(msg)
             self.unit.status = MaintenanceStatus(msg)
-            container.push("/etc/nginx/sites-available/default", nginx_config)
-            with open("files/nginx-logging-format.conf", "r", encoding="utf-8") as file:
+            container.push("/etc/nginx/sites-enabled/default", nginx_config)
+            with open("rockfile/nginx-logging-format.conf", "r", encoding="utf-8") as file:
```

Comment:
Out of curiosity, does this file need to be dynamically pushed? If not, I think we could try baking the configurations into the image directly. 👍

Reply:
The .conf file could be pasted directly into the image, but the nginx tmpl is not, since I have to able to change juju's config in real time and see the template formatted accordingly.